service/src/main/java/bio/terra/cbas/common/MicrometerMetrics.java

```diff
@@ -22,4 +23,12 @@ public void logRunCompletion(String completionTrigger, CbasRunStatus resultsStat
             .register(meterRegistry);
     counter.increment();
   }
+
+  public Timer.Sample startTimer() {
+    return Timer.start(meterRegistry);
+  }
+
+  public void stopTimer(Timer.Sample sample, String name, String... tags) {
+    sample.stop(meterRegistry.timer(name, tags));
```

Comment:
Out of curiosity, do the timer metrics also get tagged with workspace ID by default?

Reply:
looks like they do, in the `customDimensions` field